src/screens/Settings/LabeledSettingsRow.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+
+import { typography, palette } from 'app/styles';
+
+interface Props {
+  children: React.ReactNode;
+  label: string;
+}
+
+export const LabeledSettingsRow = ({ children, label }: Props) => {
+  return (
+    <View>
+      <View style={styles.greyLine}></View>
+      <Text style={styles.label}>{label}</Text>
+      {children}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  label: {
+    paddingBottom: 16,
+    ...typography.caption,
+    color: palette.textGrey,
+  },
+  greyLine: {
+    marginVertical: 24,
+    borderColor: palette.grey,
+    width: '120%',
```

Comment:
This looks shady :) at least in the code. If it's looking good in the app then it's okay

Reply:
Yup, I've didn't found another way to set children element width higher than it's parent. This element exist in `ScreenTemplate` which has `padding: 20`. Another way could be passing styles by props to `ScreenTemplate` and change the padding to 0, then make some children padding 20 and some padding 0.